backend/pkg/transport/network/tftp/tftp_test.go

```diff
@@ -0,0 +1,38 @@
+package tftp_test
+
+import (
+	"bytes"
+	"github.com/HyperloopUPV-H8/h9-backend/pkg/transport/network/tftp"
+	"testing"
+)
+
+func TestTFTP(t *testing.T) {
```

Comment:
you are missing a server in this implementation, our library only implements a client, you will need to use the underlying library (`github.com/pin/tftp/v3`) to implement the server